src/rgw/rgw_lc.cc

```diff
@@ -816,6 +817,11 @@ class LCOpAction_Transition : public LCOpAction {
     target_placement.inherit_from(oc.bucket_info.placement_rule);
     target_placement.storage_class = transition.storage_class;
 
+    if (!oc.store->svc.zone->get_zone_params().valid_placement(target_placement)) {
+      ldpp_dout(oc.dpp, 0) << "ERROR: non existent dest placement: " << target_placement << dendl;
```

Comment:
i think it would be helpful for the log message to include the bucket name and lifecycle rule name, so the admin knows which policy they need to fix

Reply:
Added, current message looks like 
`lifecycle: ERROR: non existent dest placement: default-placement/rrs bucket=foobar[675127e2-4e2c-4731-9499-129fb4295014.4317.1] rule_id=Transition abc objects`